_sources/Selection/BooleanValuesandBooleanExpressions.rst

```diff
@@ -63,10 +63,18 @@ two values are equal to one another.
 .. activecode:: ch05_2
 
     print(5 == 5)
+
     print(5 == 6)
+    
+    j = "hel" + "lo"
+    k = "hello"
+    print(j == k)
+    
 
 In the first statement, the two operands are equal, so the expression evaluates
 to ``True``.  In the second statement, 5 is not equal to 6, so we get ``False``.
+In the third statement, we see that concatenation in j wil result in the same value
```

Comment:
typo: missing an l in will

Reply:
I apologize, I should have had someone review it first. I have fixed the misspelling.